wallet/wallet.go

```diff
@@ -2382,6 +2382,88 @@ func (w *Wallet) TransactionSummary(txHash *chainhash.Hash) (*TransactionSummary
 	return txSummary, err
 }
 
+// GetTicketsResult response struct for gettickets rpc request
+type GetTicketsResult struct {
+	Tickets []*TicketSummary
+}
+
+// GetTickets implements the rpc request command for gettickets
+func (w *Wallet) GetTickets(startBlock, endBlock *BlockIdentifier, cancel <-chan struct{}) (*GetTicketsResult, error) {
+	var start, end int32 = 0, -1
+
+	if startBlock != nil {
+		if startBlock.hash == nil {
+			start = startBlock.height
+		} else {
+			err := walletdb.View(w.db, func(dbtx walletdb.ReadTx) error {
+				ns := dbtx.ReadBucket(wtxmgrNamespaceKey)
+				meta, err := w.TxStore.GetBlockMetaForHash(ns, startBlock.hash)
```

Comment:
Look up the header and extract the height from that.  I want to remove BlockMeta eventually as it is old code that is made irrelevant now that all headers are saved.